src/exceptions/clientException.ts

```diff
@@ -13,7 +13,7 @@ export class ClientException extends Exception {
 
   constructor(serviceName: string, error?: AxiosError | unknown) {
     super('Dependent service returned error', ClientException.convertStatusCode(error), {
-      error,
+      error: axios.isAxiosError(error) ? error.response?.data?.details : error,
```

Comment:
I think the `.details` is specific to an API. The `error.response.data` corresponds to the error response body from Axios, correct?

If that's correct, then only `error.response.data` makes sense to me.

Reply:
yeah, you're right, details field is not from Axios